flexget/plugins/input/from_telegram.py

```diff
@@ -0,0 +1,227 @@
+            'whitelist': {
+                'type': 'array',
+                'minItems': 1,
+                'items': {
+                    'oneOf': [
+                        {
+                            'type': 'object',
+                            'properties': {'username': {'type': 'string'}},
+                            'required': ['username'],
+                            'additionalProperties': False,
+                        },
+                        {
+                            'type': 'object',
+                            'properties': {
+                                'fullname': {
+                                    'type': 'object',
+                                    'properties': {
+                                        'first': {'type': 'string'},
+                                        'sur': {'type': 'string'},
+                                    },
+                                    'required': ['first', 'sur'],
+                                    'additionalProperties': False,
+                                }
+                            },
+                            'required': ['fullname'],
+                            'additionalProperties': False,
+                        },
+                        {
+                            'type': 'object',
+                            'properties': {'group': {'type': 'string'}},
+                            'required': ['group'],
+                            'additionalProperties': False,
+                        },
+                    ]
+                },
+            },
+            'only_new': {'type': 'boolean', 'default': True},
+            'entry': {
+                'type': 'object',
+                'properties': {
+                    'url': {'type': 'string', 'format': 'regex'},
+                    'title': {'type': 'string', 'format': 'regex'},
+                },
+                'additionalProperties': {'type': 'string', 'format': 'regex'},
+            },
+        },
+        'required': ['token'],
+        'additonalProperties': False,
+    }
+
+    @plugin.internet(logger)
+    def on_task_input(self, task, config):
+        # Load The Configs
+        token = config['token']
+        only_new = config['only_new']
+        entry_config = config.get('entry')
+        whitelist = config.get('whitelist', [])
+        types = config.get('types', ['private', 'group'])
+
+        # Get Last Checked ID
+        persistence_name = f"{token}_update_id"
+        update_id = task.simple_persistence.get(persistence_name)
+
+        # Get only new messages
+        params = {}
+        if update_id and only_new:
+            update_id += 1
+            params['offset'] = update_id
+
+        # The Target URL
+        url = f"{_TELEGRAM_API_URL}bot{token}/getUpdates"
+
+        # Get Telegram Updates
+        try:
+            response = task.requests.get(url, timeout=60, raise_status=False, params=params).json()
```

Comment:
just noticed this, but if you set raise_status=False it won't raise an exception here. you need to set it to `True` (which is the default I think)

Reply:
Done @liiight ! Tks for the time spend on this